webextension/popup.js

```diff
@@ -101,6 +101,31 @@ function getShortCode(languageCode) {
     return languageCode.replace(/-.*/, "");
 }
 
+function suggestionBackgroundStyle(match){
+    if(isSpellingError(match)){
+        return 'hiddenSpellError';
+    } else if (isSuggestion(match)){
+        return 'hiddenSuggestion';
+    } else {
+        return 'hiddenGrammarError';
+    }
+}
+
+function isSpellingError(match){
```

Comment:
this file seems to prefer a space after `(...)` and before the opening curly brace.